### PR TITLE
newone second commit

### PR DESCRIPTION
**Pull request by Kantharajn88**
verify my code and push the code into master

### CODE REVIEW

File: newone.txt [hunk 1]
```diff
@@ -1 +1,2 @@
-new cfrdghjwhFJJVJJ
```
Comment (BinduRaj05) [inline newone.txt:1]: 1st commit

File: newone.txt [hunk 2]
```diff
@@ -1 +1,2 @@
...
\ No newline at end of file
+
+Bby please verify my code and added in master branch
```
Comment (BinduRaj05) [inline newone.txt:2]: hi kanthu

Comment (BinduRaj05) [inline newone.txt:2]: hi Kanthu

Comment (Kantharajn88) [reply newone.txt:2]: i have changed

**Pull request merged**